### PR TITLE
Add documentation for `image-url-helper` components subcommand

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- Add documentation for `image-url-helper` components subcommand

**Related issue(s)**

### CODE REVIEW

File: development/image-url-helper/README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Image URL Helper is a tool that provides the following subcommands:
 * The `check` command finds all Helm chart images that don't use the `imageurl` template.
 * The `list` command lists all Helm chart images by checking the `values.yaml` files.
 * The `promote` command updates the container registry path and Helm chart images versions in the `values.yaml` files. The subcommand also prints a YAML that can be used by the [Image Syncer](../image-syncer) tool to promote images.
+* The `components` command generates [Component-Descriptor](https://github.com/gardener/component-spec) file for Kyma repository from `values.yaml` files.
```
Comment (mmitoraj) [inline development/image-url-helper/README.md:10]:
```suggestion
* The `components` command generates a [Component-Descriptor](https://github.com/gardener/component-spec) file for the `kyma` repository from `values.yaml` files.
```

File: development/image-url-helper/README.md [hunk 2]
```diff
@@ -85,3 +99,19 @@ See the list of flags available for the `promote` command:
...
 | **--excludes-list** |    No    | Path to the list of excluded `values.yaml` files.|
+
+### Components command flags
+
+See the list of flags available for the `components` command:
+
+| Name                      | Required | Description                                                                                          |
+| :------------------------ | :------: | :--------------------------------------------------------------------------------------------------- |
+| **--resources-directory** |   Yes    | Path to the Kyma resources directory.|
+| **--component-name** |    No    | Name of the component described in the `Component-Descriptor` file. Defaults to `github.com/kyma-project/kyma`.|
```
Comment (mmitoraj) [inline development/image-url-helper/README.md:110]:
```suggestion
| **--component-name** |    No    | Name of the component described in the `Component-Descriptor` file. The default value is `github.com/kyma-project/kyma`.|
```

File: development/image-url-helper/README.md [hunk 3]
```diff
@@ -85,3 +99,19 @@ See the list of flags available for the `promote` command:
...
+| **--component-version** |    Yes    | Version of the component.|
+| **--provider** |    No    | Component provider (internal or external). Defaults to `internal`.|
```
Comment (mmitoraj) [inline development/image-url-helper/README.md:112]:
```suggestion
| **--provider** |    No    | Component provider (internal or external). The default value is `internal`.|
```

File: development/image-url-helper/README.md [hunk 4]
```diff
@@ -85,3 +99,19 @@ See the list of flags available for the `promote` command:
...
+| **--git-commit** |    Yes    | Hash of the git commit.|
+| **--git-branch** |    Yes    | Name of the base git branch.|
+| **--skip-image-hashing** |    No    | The boolean value that controls image tag to hash conversion. Defaults to `false`.|
```
Comment (mmitoraj) [inline development/image-url-helper/README.md:115]:
```suggestion
| **--skip-image-hashing** |    No    | Boolean value that controls image tag to hash conversion. The default value is `false`.|
```

**Review by mmitoraj — COMMENTED** [review]
I added a few suggetsions. Thank you!

**Review by mmitoraj — APPROVED** [review]

**Review by adamwalach — APPROVED** [review]

**Pull request merged**